### ISSUE TITLE
Keep order of static and dynamic elements when mixed

### ISSUE THREAD

**Issue by olanod** [DioxusLabs/dioxus#594]
I have a container with some static headlines followed by dynamic content, until now elements were render in the order of declaration as expected but in **`master`**(on the web) after the template optimization thingy now things get mixed. 

```rust
let some_elements = SE.iter().map(|x| rsx! { "{x}" });
let extra_elements = EE.iter().map(|y| rsx! { "{y}" });
let more_elements = ME.iter().map(|z| rsx! { "{z}" });

render! {
  div {
    some_elements,
    h2 { "Extra elements" },
    extra_elements,
    h2 { "More elements" },
    more_elements,
  }
}
```

Now I get `some_elements`, `extra_elements` and `more_elements` mixed together with the static headlines rendered as the last children of the container.

**Comment by Demonthos** [comment]
This was just fixed on master (#591). I tried the example and it seems to work now. You might need to run ```cargo update``` to get the latest patched version.

**Comment by olanod** [comment]
It works :)
